app/assets/scripts/components/Header.js

```diff
@@ -9,6 +9,9 @@ class Header extends Component {
   static propTypes = {
     history: PropTypes.object.isRequired
   }
+  componentWillMount() {
+    document.title = 'Historic Disaster Explorer';
```

Comment:
Any specific reason why you're setting the title here, instead of in the `index.html`?

Reply:
I just corrected the observation, thanks so much @danielfdsilva. :bowing_man: